static/src/javascripts/es6/projects/common/modules/crosswords/main.js

```diff
@@ -416,6 +417,21 @@ class Crossword extends React.Component {
         this.save();
     }
 
+    onClearSingle () {
+        const cellsInFocus = helpers.cellsForEntry(this.clueInFocus());
+
+        this.setState({
+          grid: helpers.mapGrid(this.state.grid, (cell, gridX, gridY) => {
+              if (_.some(cellsInFocus, cell => cell.x === gridX && cell.y === gridY)) {
```

Comment:
I'm, not sure, i think it's possible to be considered modern and not have `some`? https://github.com/guardian/frontend/blob/master/common/app/templates/headerInlineJS/config.scala.js#L8